selftests/unit/plugin/test_resolver.py

```diff
@@ -113,16 +113,15 @@ def test_exec_test(self):
         with script.TemporaryScript('exec-test.sh', "#!/bin/sh\ntrue",
                                     'test_resolver_exec_test') as exec_test:
             res = ExecTestResolver().resolve(exec_test.path)
-            self.assertEqual(res.reference, exec_test.path)
-            self.assertEqual(res.result, resolver.ReferenceResolutionResult.SUCCESS)
-            self.assertEqual(len(res.resolutions), 1)
-            resolution = res.resolutions[0]
-            self.assertEqual(resolution.kind, 'exec-test')
-            self.assertEqual(resolution.uri, exec_test.path)
-            self.assertEqual(resolution.args, ())
-            self.assertEqual(resolution.kwargs, {})
-            self.assertEqual(resolution.tags, None)
-
+        self.assertEqual(res.reference, exec_test.path)
+        self.assertEqual(res.result, resolver.ReferenceResolutionResult.SUCCESS)
+        self.assertEqual(len(res.resolutions), 1)
+        resolution = res.resolutions[0]
+        self.assertEqual(resolution.kind, 'exec-test')
+        self.assertEqual(resolution.uri, exec_test.path)
+        self.assertEqual(resolution.args, ())
+        self.assertEqual(resolution.kwargs, {})
+        self.assertEqual(resolution.tags, None)
```

Comment:
I don't know why, but I have these lines showing they were changed, but I don't see any change. This is the case for this commit.

Reply:
They were just re-indented... not sure if that's the reason.

Reply:
Ah, interesting, it is not even showing the indentation here.